api/api.go

```diff
@@ -2,23 +2,27 @@ package api
 
 import (
 	"context"
+	"net/http"
 
 	"github.com/ONSdigital/log.go/log"
 	"github.com/gorilla/mux"
 )
 
 //API provides a struct to wrap the api around
 type API struct {
-	Router *mux.Router
+	Router  *mux.Router
+	mongoDB PermissionsStore
```

Comment:
Consider also calling the variable `permissionsStore` - for consistency, and resistant against a change in the data store.
```suggestion
	permissionsStore PermissionsStore
```